addons/cert-manager/resources/parameter.cue

```diff
@@ -1,3 +1,6 @@
 parameter: {
+  // +usage=Specify the targetNamespace where the cert-manager component will apply
+  "tgtNs": *"cert-manager"| string
```

Comment:
If you want a namespace, you need to create it first (there are many ways to do this), otherwise it won't install.